test/support/api/workspace-patients.js

```diff
@@ -1,12 +1,16 @@
 import _ from 'underscore';
-import { getResource } from 'helpers/json-api';
+import { getResource, mergeJsonApi } from 'helpers/json-api';
 
 import fxWorkspacePatients from 'fixtures/collections/workspace-patients';
 
 const TYPE = 'workspace-patients';
 
-export function getWorkspacePatient() {
-  return getResource(_.sample(fxWorkspacePatients), TYPE);
+export function getWorkspacePatient(data) {
+  const resource = getResource(_.sample(fxWorkspacePatients), TYPE);
+
+  if (!data) return resource;
```

Comment:
I don't think we want this guard here.  `mergeJSON` also clones the object so it can't be mutated for future tests

Reply:
Removed 👍